capsules/src/process_console.rs

```diff
@@ -34,7 +34,7 @@ pub const READ_BUF_LEN: usize = 4;
 /// characters, limiting arguments to 25 bytes or so seems fine for now.
 pub const COMMAND_BUF_LEN: usize = 32;
 
-pub const DEFAULT_COMMAND_HISTORY_LEN: usize = 10;
+pub const DEFAULT_COMMAND_HISTORY_LEN: usize = 11;
```

Comment:
Keep this 10 and add to the documentation that this includes the current command.